From 1539f4fd6b80738ace8ef35b36384dedbeea9f20 Mon Sep 17 00:00:00 2001
From: John Ogness <john.ogness@linutronix.de>
Date: Tue, 22 Nov 2022 10:01:23 +0106
Subject: [PATCH 59/59] printk: htmldocs: add missing description

Variable and return descriptions were missing from the SRCU read
lock functions. Add them.

Signed-off-by: John Ogness <john.ogness@linutronix.de>
Signed-off-by: Petr Mladek <pmladek@suse.com>
Link: https://lore.kernel.org/r/87zgcjpdvo.fsf@jogness.linutronix.de
---
 kernel/printk/printk.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index a40dd138d..7decf1e9c 100644
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -282,6 +282,7 @@ EXPORT_SYMBOL(console_list_unlock);
  * Use for_each_console_srcu() to iterate the console list
  *
  * Context: Any context.
+ * Return: A cookie to pass to console_srcu_read_unlock().
  */
 int console_srcu_read_lock(void)
 {
@@ -292,6 +293,7 @@ EXPORT_SYMBOL(console_srcu_read_lock);
 /**
  * console_srcu_read_unlock - Unregister an old reader from
  *	the SRCU-protected console list
+ * @cookie: cookie returned from console_srcu_read_lock()
  *
  * Counterpart to console_srcu_read_lock()
  */
-- 
2.38.1.473.ga0789512c5

